src/sensor.cpp

```diff
@@ -509,10 +507,9 @@ void log_callback_end( uint32_t fps,
                     diff = environment::get_instance().get_time_service()->get_time() - system_time;
                     if (diff >10 )
                         LOG_DEBUG("!! Frame memcpy took " << diff << " msec");
-                    if (!requires_processing)
-                    {
-                        fh->attach_continuation(std::move(release_and_enqueue));
-                    }
+
+                    // calling the continuation method, and releasing the backend frame buffer
```

Comment:
Please explain:
`// since we've copied the content of the OS frame buffer, we can release it ASAP`

Reply:
We do memcpy and keep the frame inside the SDK.
From this point on we do not need the OS buffer to be locked and we can release it.

More information inside the ticket

Reply:
I know, I want it to say that in the code, the current comment is not very helpful

Reply:
done